etc/profile-m-z/vlc.profile

```diff
@@ -8,6 +8,7 @@ include globals.local
 
 noblacklist ${HOME}/.cache/vlc
 noblacklist ${HOME}/.config/vlc
+noblacklist ${HOME}/.config/aacs
```

Comment:
There is no such blacklist. Either remove this, or better add a blacklist statement to disable-programs.inc.

Reply:
Thanks! I will add the path to disable.programs.inc